src/gradient.rs

```diff
@@ -69,7 +69,7 @@ impl ColorStop {
         since = "0.2.0",
         note = "This method has been renamed to `multiply_alpha`."
     )]
-    pub fn with_alpha_factor(self, alpha: f32) -> Self {
+    pub const fn with_alpha_factor(self, alpha: f32) -> Self {
```

Comment:
We probably should remove this before doing a release (not for this PR)

Reply:
I'm fine with this sticking around for the 0.3 release cycle and removing it in 0.4

Reply:
I'm not sure I see the advantage, since to make sure we actually do this, we should do it imminently (i.e. immediately after the release)

But at that point, why not just do it now? It should be a very small change.

Reply:
Okay, will just submit a PR that removes the deprecated stuff now.

Reply:
PR #82 removes it.